engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -64,6 +80,13 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
   }
 
   when(WorkflowExecutionInProgressState) {
+    case Event(BackendJobExecutionSucceededResponse(jobKey, callOutputs), stateData) =>
+      log.info(s"Job ${jobKey.call.fullyQualifiedName} succeeded! Outputs: ${callOutputs.mkString(" |:::| ")}")
+      context.parent ! WorkflowExecutionSucceededResponse
+      goto(WorkflowExecutionSuccessfulState)
+    case Event(BackendJobExecutionFailedResponse(jobKey, reason), stateData) =>
+      log.warning(s"Job ${jobKey.call.fullyQualifiedName} succeeded! Reason: $reason")
```

Comment:
Hmm this looks more like a wrong copy/paste, should be `failed` instead of `succeeded` here